internal/controllers/certmanager.go

```diff
@@ -131,6 +164,37 @@ func (r *Reconciler) setupTLS(ctx context.Context, cr *model.CryostatInstance) (
 	return tlsConfig, nil
 }
 
+func (r *Reconciler) finalizeTLS(ctx context.Context, cr *model.CryostatInstance) error {
+	caCert := resources.NewCryostatCACert(cr)
+	err := r.createOrUpdateCertificate(ctx, caCert, cr.Object)
+	if err != nil {
+		return err
+	}
```

Comment:
This part can be removed, we don't want to create/update the certificate here since it's in the process of being deleted.